web/cypress/integration/consumer/create-consumer-with-proxy-mirror-form.spec.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create and Delete Consumer', () => {
```

Comment:
What is the relationship between this plugin and consumer？

Reply:
1. All Plug-ins in Consumers need to configured, but in other entities like Service/Route, Auth plugins don't need to be configured, only enable or disable.
2. To improve the Test case coverage as full as possible, we choose to test plugins in Consumer, we need to configure at least one Auth plugin and target plugin.
3. Do you mean only need test the target plugin component? It's more like a Single Component test.

Reply:
We need to test the setting of plugins in routing, which is the most commonly used

Reply:
Got it.